featurecontrol/featurecontrol.go

```diff
@@ -70,9 +107,17 @@ func NewFlags(logger log.Logger, features string) (Flagger, error) {
 		opt(fc)
 	}
 
+	if fc.stableMatchersParsing && fc.utf8MatchersParsing {
```

Comment:
@gotjosh this seemed like the most sensible thing to do here. I wasn't keen on having an implicit link between both `"stable-matchers-parsing"` and `"utf8-matchers-parsing"` being enabled and using the fallback parser.